platform_versioned_docs/version-24.1.0/enterprise/_templates/docker/tower.env

```diff
@@ -33,7 +33,6 @@ GROUNDSWELL_SERVER_URL="http://groundswell:8090"
 #TOWER_DATA_EXPLORER_ENABLED=true
 #TOWER_DATA_EXPLORER_CLOUD_DISABLED_WORKSPACES=
 #
-#TOWER_DATA_STUDIO_CONNECT_URL=<YOUR CONNECT URL, SAME AS `CONNECT_PROXY_URL` IN `data-studios.env` >
-#TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN=<YOUR INITIAL ACCESS TOKEN, IDEANTICAL AS `INITIAL_ACCESS_TOKEN` in `data-studios.env`>
+#TOWER_DATA_STUDIO_CONNECT_URL=<YOUR CONNECT URL, SAME AS `CONNECT_PROXY_URL` IN `data-studios.env`>
+#TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN=<use the same OIDC_CLIENT_REGISTRATION_TOKEN as in data-studios.env>
```

Comment:
We should also add this commented section for data studios
```
TOWER_DATA_STUDIO_TEMPLATES_JUPYTER_REPOSITORY: 'cr.seqera.io/public/data-studio-jupyter:rc'
TOWER_DATA_STUDIO_TEMPLATES_JUPYTER_ICON: 'jupyter'
TOWER_DATA_STUDIO_TEMPLATES_JUPYTER_TOOL: 'jupyter'
TOWER_DATA_STUDIO_TEMPLATES_JUPYTER_PORT: '8080'
TOWER_DATA_STUDIO_TEMPLATES_RSTUDIO_REPOSITORY: 'cr.seqera.io/public/data-studio-rstudio:rc'
TOWER_DATA_STUDIO_TEMPLATES_RSTUDIO_ICON: 'rstudio'
TOWER_DATA_STUDIO_TEMPLATES_RSTUDIO_TOOL: 'rstudio'
TOWER_DATA_STUDIO_TEMPLATES_RSTUDIO_PORT: '8787'
TOWER_DATA_STUDIO_TEMPLATES_VSCODE_REPOSITORY: 'cr.seqera.io/public/data-studio-vscode:rc'
TOWER_DATA_STUDIO_TEMPLATES_VSCODE_ICON: 'vscode'
TOWER_DATA_STUDIO_TEMPLATES_VSCODE_TOOL: 'vscode'
TOWER_DATA_STUDIO_TEMPLATES_VSCODE_PORT: '3000'
```
@endre-seqera does kubernetes setup use env vars too?

Reply:
If these are going to be listed, I think some additional info will be required re: the ports (i.e. is it ok to change? Hardcoded? If they change must they change something else? Do we expect these to change?)

While I generally like to publish most content externally, I wonder if these settings are not one of them (unless the idea is that Enterprise customers could implement additional templates following this pattern).

Reply:
@t0randr are all of those variables required? Or maybe just the repository as It states the version of clients to use by platform

Reply:
@weronikasosnowskaseqera, can a user not run any Data Studios without this? Is this how a user gets "working" images to run? Thanks!

Reply:
@jason-seqera without those settings they won't have any templates of studios available so they will see a tab to create a studio but they won't be able to actually create them. They would see greyed option like this:

![image](https://github.com/seqeralabs/docs/assets/127099299/7a63668a-53e3-4c34-90ee-7208cbcacdd9)

Reply:
@weronikasosnowskaseqera the required ones are those that ends in `_REPOSITORY`.

Reply:
@weronikasosnowskaseqera - good point. The kubernetes installation also uses these variable, they need to added into the configmap (`config.yaml`) in the platform manifests.